provider/standalone/standalone.go

```diff
@@ -38,14 +38,20 @@ func (p *StandaloneProvider) ConfigureRun(state *run.State) error {
 		"cloud": "standalone",
 	}
 
-	state.ProviderMetadata = map[string]string{}
-
 	if workerLocation, ok := p.runnercfg.Provider.Data["workerLocation"]; ok {
 		for k, v := range workerLocation.(map[string]string) {
 			state.WorkerLocation[k] = v
 		}
 	}
 
+	state.ProviderMetadata = map[string]string{}
```

Comment:
I think this should be `map[string]interface{}{}` for properties that are `int` or `bool` etc.

Reply:
OK, I suppose people could include non-string values in the config file.